### PR TITLE
Update index.qmd

### PR DESCRIPTION
**Pull request by HerveCaumont**
Details about EOAP

### CODE REVIEW

File: propagation/ogc_processes.md [hunk 1]
```diff
@@ -1,33 +1,25 @@
-# OGC API Processes: file-based processing for existing software
+# OGC Application Package - file-based processing for existing software
 
-The OGC API processes specification allows you to expose basically any type of (processing) web service. As a service
-designer, you get the full freedom and responsibility in the definition of your service.
+Another option to create an APEx-compliant service is to package your software using Docker containers. This approach is very generic and is particularly relevant if you have an existing piece of complex software that you want to make available as a service.
```
Comment (jdries) [inline propagation/ogc_processes.md:3]: Please clarify better web service vs Docker container.
The end result that we need according to SoW is an on-demand web service, so based on OGC API Processes.
OGC Application package deployed on hosting platform is the recommended approach to reach that goal. Avoiding confusion is key here!

Then there's also the question: are projects or hosting platforms also 'allowed' to just provide the service directly, for instance without docker container?

File: propagation/ogc_processes.md [hunk 2]
```diff
@@ -1,33 +1,25 @@
...
+This process adheres to the Best Practices for Earth Observation Application Packaging as defined by the Open Geospatial Consortium (OGC) and the EO Exploitation Platform Common Architecture (EOEPCA) spearheaded by the European Space Agency (ESA). These guidelines ensure that EO applications are reproducible and portable across different execution scenarios.
```
Comment (jdries) [inline propagation/ogc_processes.md:4]: General comment: target audience are EO researchers that do not necessarily have an IT background. Lets avoid technical terms (docker, execution scenarios), or make sure to explain it, as the original text was trying to.

'execution scenarios' => should we say 'hosting platforms'? The execution scenarios are abstracted away by a compliant hosting platform anyway, so for the target audience, it perhaps makes more sense to keep common terminology. (Hosting platforms is also what we use in ICG.)

File: propagation/ogc_processes.md [hunk 3]
```diff
@@ -1,33 +1,25 @@
...
-
-To offer your application as an on-demand service, you will need to define clear constraints on possible input parameters,
-and make sure that your software is well tested to be able to handle variations in input parameters. When changes occur for 
-instance to raw EO data sources, you will need to update your software accordingly, or it might no longer be compatible with provided inputs.
-Likewise, if improvements are made to dependencies of your software, you will have the choice of updating your software, or 
-sticking to the old versions.
+The applications can be written in a variety of coding languages (e.g. Python, C++, R, Java, C#, and also shell scripts) and make use of specific software libraries, like the Geospatial Data Abstraction Library (GDAL), the Sentinel Application platform (SNAP), Orfeo Toolbox (OTB). Overall, an EO application is treated as as a black-box that needs to be:
 
+* Executable as a command-line tool.
+* Delivered in a container image with all the necessary software, libraries and configuration files.
```
Comment (jdries) [inline propagation/ogc_processes.md:12]: What about input data handling? Should it be able to read it from S3, should it be ready to support multiple formats?

File: propagation/ogc_processes.md [hunk 4]
```diff
@@ -1,33 +1,25 @@
...
-
-If concepts like containers are already familiar to you, and you prefer to have full control and responsibility over your software,
-including all details with respect to how you read raw EO products, then this might be the right choice for your project. 
-Of course, the APEx propagation services exist to help you with the integration of your service into the APEx ecosystem.
-
-To offer your application as an on-demand service, you will need to define clear constraints on possible input parameters,
-and make sure that your software is well tested to be able to handle variations in input parameters. When changes occur for 
-instance to raw EO data sources, you will need to update your software accordingly, or it might no longer be compatible with provided inputs.
-Likewise, if improvements are made to dependencies of your software, you will have the choice of updating your software, or 
-sticking to the old versions.
```
Comment (jdries) [inline propagation/ogc_processes.md:27]: This was offering some useful information, was it incorrect?  Feel free to rewrite, but it's relevant to highlight these aspects, as this is a key difference compared to the other approach.

File: propagation/ogc_processes.md [hunk 5]
```diff
@@ -1,33 +1,25 @@
...
+The applications can be written in a variety of coding languages (e.g. Python, C++, R, Java, C#, and also shell scripts) and make use of specific software libraries, like the Geospatial Data Abstraction Library (GDAL), the Sentinel Application platform (SNAP), Orfeo Toolbox (OTB). Overall, an EO application is treated as as a black-box that needs to be:
 
+* Executable as a command-line tool.
+* Delivered in a container image with all the necessary software, libraries and configuration files.
 
+Complex workflows can be built on top of the applications following fan-in and fan-out design patterns that can be combined in the nodes of a Directed Acyclic Graph (DAG).  The Application Package document ensures the EO application reproducibility and portability of the application across various execution scenarios.
 
+## Building an OGC pplication package based service
```
Comment (jdries) [inline propagation/ogc_processes.md:16]:
```suggestion
## Building an OGC application package based service
```

File: propagation/ogc_processes.md [hunk 6]
```diff
@@ -1,33 +1,25 @@
...
-instance to raw EO data sources, you will need to update your software accordingly, or it might no longer be compatible with provided inputs.
-Likewise, if improvements are made to dependencies of your software, you will have the choice of updating your software, or 
-sticking to the old versions.
+The applications can be written in a variety of coding languages (e.g. Python, C++, R, Java, C#, and also shell scripts) and make use of specific software libraries, like the Geospatial Data Abstraction Library (GDAL), the Sentinel Application platform (SNAP), Orfeo Toolbox (OTB). Overall, an EO application is treated as as a black-box that needs to be:
 
+* Executable as a command-line tool.
+* Delivered in a container image with all the necessary software, libraries and configuration files.
 
+Complex workflows can be built on top of the applications following fan-in and fan-out design patterns that can be combined in the nodes of a Directed Acyclic Graph (DAG).  The Application Package document ensures the EO application reproducibility and portability of the application across various execution scenarios.
```
Comment (jdries) [inline propagation/ogc_processes.md:14]: portability => also with respect to input data? Is that a responsibility of the AP builder, or the hosting platform?

**Review by jdries — CHANGES_REQUESTED** [review]

File: propagation/ogc_processes.md (hunk 4 above)
Comment (JanssenBrm) [reply propagation/ogc_processes.md:27]: To be reviewed by @HerveCaumont or @p3dr0  - This could be due to my merge of the content.

**Pull request closed without merging**